web/src/lib/components/assets/thumbnail/thumbnail.svelte

```diff
@@ -241,7 +241,7 @@
           class="absolute z-30 {className} top-[41px]"
```

Comment:
What happens if you do this?
```suggestion
          class="absolute z-30 {className} top-0"
```

Reply:
From what I tested previously, this offset is needed, otherwise you can't click on the checkmark to actually start the selection process.

Reply:
Oh, I figured out a solution.
Change `z-30` to `z-20` and on the div below change `z-20` to `z-30` and you can then set 0 in my suggestion.